messaging/kafka-producer/src/test/java/io/quarkus/ts/messaging/kafka/producer/SnappyCompressionIT.java

```diff
@@ -78,14 +80,15 @@ public void checkSnappyCompressionSize() throws IOException, InterruptedExceptio
     @Test
     public void checkIntegrityMessageAfterCompression() {
         String msg = "This is the message";
-        KafkaConsumer<Integer, String> consumer = createConsumer();
-        UniAssertSubscriber<Object> subscriber = makeHttpReqWithMessage("/messageEvent", msg)
-                .subscribe().withSubscriber(UniAssertSubscriber.create());
-        subscriber.awaitItem(Duration.ofSeconds(TIMEOUT_SEC)).getItem();
-
-        ConsumerRecord<Integer, String> records = consumer.poll(Duration.ofMillis(10000))
-                .iterator().next();
-        Assertions.assertEquals(records.value(), msg);
+        try (var consumer = createConsumer()) {
```

Comment:
Good approach with try-with-resources, ensuring the closure of consumer. Thanks.